src/jobBundle/Controller/JobController.php

```diff
@@ -0,0 +1,189 @@
+    {
+
+        $repository = $this->getDoctrine()
+                           ->getRepository(jobEntity::class)
+                           ->findAll();
+
+      return $this->render('job/job_select.html.twig', array(
+            'jobs' => $repository
+      ));
+    }
+    
+    // The user can add new job types to the database.
+
+     /**
+     * @Route("job/job_new")
+     */
+    public function addJob(Request $request)
+    {
+        $job = new jobEntity();
+
+       $form = $this->createFormBuilder()
+        ->add('name', TextType::class)
+        ->add('Felvesz', SubmitType::class)
+        ->getForm();
+       
+        $form->handleRequest($request);
+        
+        if($form->isSubmitted() && $form->isValid()){
+            
+            $name = $form['name']->getData();
+            
+            $job->setName($name);
+            
+             $em = $this->getDoctrine()->getManager();
+             
+             $em->persist($job);
+             $em->flush();
+             
+             $this->addFlash(
+               'notice',
+               'Job added'
+            );
+         
+         return $this->redirectToRoute("job_job_getjobs");
+        }
+        
+        return $this->render('job/job_new.html.twig', array(
+            'form' => $form->createView()
+        ));
+        
+    }
+    
+    // Find a job by its id, if it is already in database
+    
+     /**
+     * @Route("job/job_search")
+     */
+    
+    public function getJob(Request $request)
+    {
+       $job = new jobEntity();
+
+       $form = $this->createFormBuilder()
+        ->add('id', TextType::class)
+        ->add('Keres', SubmitType::class)
+        ->getForm();
+       
+        $form->handleRequest($request);
+        
+        if($form->isSubmitted() && $form->isValid()){
+            
+            $id = $form['id']->getData();
+            
+            $job = $this->getDoctrine()
+           ->getRepository(jobEntity::class)
+           ->find($id);
+            
+           if (!$job) {
+                return new Response('Nincs ilyen id, hogy '.$id );
+              } else {
+                 $job_name = $job->getName();
+                return new Response('Van ilyen id , '. $id. ' Neve: '. $job_name   );
+           }
+        
+        }
+        
+        return $this->render('job/job_search.html.twig', array(
+            'form' => $form->createView()
+         ));
+            
+    }
+    
+     // Select all of the jobs, and select for update the name of the job
+    
+      /**
+     * @Route("job/job_update")
+     */
+
+    public function getJobsForUpdate(Request $reguest)
+    {
+ 
+          $repository = $this->getDoctrine()
+                           ->getRepository(jobEntity::class)
+                           ->findAll();
+          
+         return $this->render('job/job_update.html.twig', array(
+            'jobs' => $repository
+          ));
+    }
+    
+    // Update the job name 
```

Comment:
Ezt a kommentet vidd be a javaDoc első sorába!
/**
 * ezt csinálja a függvény
 *
 * dokumentáció ...
 **/